packages/core/src/codewhispererChat/controllers/chat/controller.ts

```diff
@@ -771,7 +771,7 @@ export class ChatController {
                         customization: getSelectedCustomization(),
                         toolResults: toolResults,
                         origin: Origin.IDE,
-                        context: session.context ?? [],
```

Comment:
setting falcon context here in the toolResults response will waste tokens, hence removing it